src/Storage/Commands.Storage/Azure.Storage.psd1

```diff
@@ -57,7 +57,11 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.4.0'; }
 RequiredAssemblies = '.\Microsoft.WindowsAzure.Storage.dll', 
                '.\Microsoft.WindowsAzure.Storage.DataMovement.dll', 
                '.\Microsoft.Azure.KeyVault.Core.dll', 
-               '.\Microsoft.WindowsAzure.Management.dll'
+               '.\Microsoft.WindowsAzure.Management.dll', 
+               '.\Microsoft.Azure.ResourceManager.dll', 
```

Comment:
ResourceManager dll is used here?

Reply:
We can check this post-merge.

Reply:
Added issue for it: https://github.com/Azure/azure-powershell/issues/6952